src/packer.pkr.hcl

```
@@ -142,13 +151,185 @@ build {
   }
 
   provisioner "powershell" {
-    # Wait 90 seconds before executing the check-defender.ps1 powershell script.
-    # This gives a generous grace period between restarting Windows and running the second provisioner.
-    pause_before = "90s"
-    scripts      = ["src/powershell/check-defender.ps1"]
+    # Wait 30 seconds before executing the next provisioner.
+    # This gives a grace period between restarting Windows and running the next provisioner.
+    pause_before = "30s"
+    scripts = [
+      "src/powershell/check-defender.ps1",
+      "src/powershell/enable-rdp.ps1",
+      "src/powershell/initialize-volume.ps1",
+      "src/powershell/install-chocolatey.ps1"
+    ]
+  }
+
+  provisioner "powershell" {
+    # Create "packages" directory before uploading them in the next provisioner.
+    inline = ["mkdir D:\\packages"]
+  }
+
+  provisioner "file" {
+    # Upload package lists to the "packages" directory.
+    source      = "src/packages/"
+    destination = "D:\\packages"
+  }
+
+  provisioner "powershell" {
+    # Install general packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=general"]
```

We prefer to alphabetize whenever possible:
```suggestion
    environment_vars = ["CATEGORY=general"]
    script           = "src/powershell/install-category.ps1"
```

---

@dav3r resolved with [504430a](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/504430a928e07070bb27846baaea48cf51db677a)